### ISSUE TITLE
Can't assign tuple containing a type + a TypeVarTuple to bare tuple

### ISSUE THREAD

**Issue by kltws** [microsoft/pyright#7129]
A tuple of one type + a `TypeVarTuple` can't be assigned to a bare tuple: 
(In Pyright Playground)
```python
def accepts_any_tuple(t: tuple):
    ...

def test[*_Ts](t: tuple[int, *_Ts]):
    accepts_any_tuple(t)
``` 
Argument of type "tuple[int, *_Ts@test]" cannot be assigned to parameter "t" of type "tuple[Unknown, ...]" in function "accepts_any_tuple"
  "tuple[int, *_Ts@test]" is incompatible with "tuple[Unknown, ...]"
    Tuple size mismatch; expected 1 but received 2

It looks like the tuple is inferred to have size 2 for some reason?

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.349, which I just published.
